utils/neural_net.py

```diff
@@ -0,0 +1,59 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class Net(nn.Module):
+    def __init__(self):
+        """Net constructor.
+
+        Cubes are considered of size 20 x 20 x 20 and
+        as described in the paper, we use 20 filters.
+        """
+        super(Net, self).__init__()
+
+        # Output classes
+        self.o = 14
+
+        # Number of channels (3 for RGB)
```

Comment:
```suggestion
        # Number of channels (4 for RGB)
```

Reply:
We need 1 channel for the points and 3 for the colors

Reply:
Absolutely!